src/base/rss/rssdownloadrule.h

```diff
@@ -88,6 +88,9 @@ namespace Rss
         bool operator==(const DownloadRule &other) const;
 
     private:
+        bool matches(const QString &articleTitle, QString expression) const;
```

Comment:
Hmm - I'm sure it was failing to compile at one point, but it's working now ... maybe I had another bug in there at the time.